### PR TITLE
Use loaders.gl/images for image loading

### PR DESCRIPTION
**Pull request by kylebarron**
It looks like existing code forces loading as an `ImageBitmap`. As of loaders.gl 2.1, `ImageBitmap` loading should work reliably, right? Also, loading with `load(url, ImageLoader)` should fall back to non-`ImageBitmap` on non-Chrome/Firefox, right?

Any other reason to keep the copied code?

### CODE REVIEW

File: modules/earthengine-layers/src/earth-engine-layer.js [hunk 1]
```diff
@@ -113,9 +112,7 @@ export default class EarthEngineLayer extends CompositeLayer {
         id: this._getLayerId(getTileUrl),
         async getTileData({x, y, z}) {
           const imageUrl = getTileUrl(x, y, z);
-          const image = await loadImageBitmap(imageUrl);
-          // const imageData = await getImageData(image);
-
+          const image = await load(imageUrl, ImageLoader);
```
Comment (ibgreen) [inline modules/earthengine-layers/src/earth-engine-layer.js:115]: The fast `ImageBitmap` loading path is still not the default, we need to provide an option to enable it. You need to check the docs but think it is `load(imageUrl, ImageLoader, {image: {type: 'imagebitmap'}})`.

**Comment by kylebarron** [comment]
> update loaders.gl to use `imagebitmap` by default when `auto` is true

Oh I thought that was existing behavior. So `auto` defaults to `Image`?

File: modules/earthengine-layers/src/earth-engine-layer.js (hunk 1 above)
Comment (kylebarron) [reply modules/earthengine-layers/src/earth-engine-layer.js:115]: Will setting `type` still handle fallbacks correctly?

Comment (ibgreen) [reply modules/earthengine-layers/src/earth-engine-layer.js:115]: No as indicated above, I don't think it will right now.

**Comment by ibgreen** [comment]
> Oh I thought that was existing behavior. So auto defaults to Image?

Yes deck.gl only got solid support for ImageBitmap in 8.0, so we have perhaps been overly careful to turn this on.

**Pull request merged**